### PR TITLE
[de] update links

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/32856/files
/assign @bene2k1

### CODE REVIEW

File: content/de/includes/default-storage-class-prereqs.md [hunk 1]
```diff
@@ -1 +1 @@
-Sie benötigen entweder einen dynamischen PersistentVolume-Anbieter mit einer [Standard-Speicherklasse](/docs/concepts/storage/storage-classes/), oder Sie selbst stellen statische [PersistentVolumes](/docs/user-guide/persistent-volumes/#provisioning) bereit, um die [PersistentVolumeClaims](/docs/user-guide/persistent-volumes/#persistentvolumeclaims) zu erfüllen, die hier verwendet werden.
\ No newline at end of file
+Sie benötigen entweder einen dynamischen PersistentVolume-Anbieter mit einer [Standard-Speicherklasse](/docs/concepts/storage/storage-classes/), oder Sie selbst stellen statische [PersistentVolumes](/docs/concepts/storage/user-guide/persistent-volumes/#provisioning) bereit, um die [PersistentVolumeClaims](/docs/concepts/storage/user-guide/persistent-volumes/#persistentvolumeclaims) zu erfüllen, die hier verwendet werden.
```
Comment (jihoon-seo) [inline content/de/includes/default-storage-class-prereqs.md:1]: Let's update links to the working ones.

```suggestion
Sie benötigen entweder einen dynamischen PersistentVolume-Anbieter mit einer [Standard-Speicherklasse](/docs/concepts/storage/storage-classes/), oder Sie selbst stellen statische [PersistentVolumes](/docs/concepts/storage/persistent-volumes/#provisioning) bereit, um die [PersistentVolumeClaims](/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) zu erfüllen, die hier verwendet werden.
```

**Review by jihoon-seo — COMMENTED** [review]
/label tide/merge-method-squash

@Arhell PTAL!

File: content/de/includes/default-storage-class-prereqs.md (hunk 1 above)
Comment (Arhell) [reply content/de/includes/default-storage-class-prereqs.md]: Done thanks

**Comment by rlenferink** [comment]
/lgtm
/approve

thanks for fixing

**Pull request merged**